src/openforms/js/components/admin/form_design/variables/VariablesTable.js

```diff
@@ -44,11 +69,16 @@ const VariableRow = ({index, variable}) => {
     return '';
   };
 
+  const hasErrors = !!Object.entries(variable.errors || {}).length;
```

Comment:
this `Object.entries(objectLike || {}).length` is repeated quite often - can we check if `lodash` has something for this, and otherwise put it in a utility?